wire-ios/Configuration/Developer-Flags.xcconfig

```diff
@@ -17,7 +17,7 @@
 //
 
 
-PROTEUS_BY_CORECRYPTO_ENABLED=0
-MLS_ENABLED=0
+PROTEUS_BY_CORECRYPTO_ENABLED=1
+MLS_ENABLED=1
 // Shows the toggle to choose MLS as messaging protocol when creating a group
-CREATE_MLS_GROUP_ENABLED=0
+CREATE_MLS_GROUP_ENABLED=1
```

Comment:
should be reverted

Reply:
Just `CREATE_MLS_GROUP_ENABLED`?

Reply:
no the full file, as develop should not have those flags on

Reply:
reverted all flags